src/components/ScholarshipListCard.jsx

```diff
@@ -29,7 +29,11 @@ function ScholarshipListCard({ scholarship }) {
           state: { scholarship },
         }}>
         <CardContent sx={{ p: 3 }}>
-          <Typography variant="subtitle1" sx={{ fontWeight: '500' }}>
+          <Typography
+            variant="subtitle1"
+            sx={{
+              fontWeight: (theme) => theme.typography.fontWeightMedium,
```

Comment:
Per [docs](https://mui.com/system/the-sx-prop/#typography) you can do:
```suggestion
              fontWeight: 'medium',
```